rpmsign.c

```diff
@@ -149,9 +170,9 @@ static int doSign(poptContext optCon, struct rpmSignArgs *sargs)
 
     if (flags_sign_files(sargs->signflags)) {
 	char *fileSigningKeyPassword = NULL;
-	char *key = rpmExpand("%{?_file_signing_key}", NULL);
```

Comment:
I'm concerned this is not backward compatible. We can only consider removing a feature if it's never used because it's broken. Is this broken?

Reply:
Yeah it is. 

The signing actually calls "`libfsverity_sign_digest`" (from kernel fsverity-utils: https://git.kernel.org/pub/scm/linux/kernel/git/ebiggers/fsverity-utils.git/tree/lib/sign_digest.c#n402) in "`sign/rpmsignverity.c`", which has requirement on 
- cert present, 
- private key can either from direct private key path or PKCS#11 token, but must be present as well. 

So this feature is not removed, it's now
- private key can be provided in two ways (follow fsverity-utils), 
- cert being present check performed earlier to fail early if not present.